examples/seismic/skew_self_adjoint/operators.py

```diff
@@ -97,56 +87,45 @@ def IsoFwdOperator(model, src, rec, time_axis, space_order=8, save=False, **kwar
     ----------
     The Operator implementing forward modeling.
     """
-    # Get the Functions for buoyancy, velocity, wOverQ
-    assert 'b' in model, "model dictionary must contain key 'b'"
-    assert 'v' in model, "model dictionary must contain key 'v'"
-    assert 'wOverQ' in model, "model dictionary must contain key 'wOverQ'"
-    b = model['b']
-    v = model['v']
-
+    src = geometry.src
+    rec = geometry.rec
+    vp, b = model.vp, model.b
     # Create symbols for wavefield, source and receivers
-    u = TimeFunction(name='u', grid=v.grid,
-                     save=time_axis.num if save else None,
+    u = TimeFunction(name='u', grid=model.grid,
+                     save=geometry.num if save else None,
```

Comment:
hmmm.. I would like more save to be assicated with something that has to do **more** with time.

Reply:
In any seismic setup, the data and therefore the geometry defines the time/shape/... so makes sense to get it from there. Remember tht this is in the operator creation so these are just placeholder not the actual runtime objects (most of the time)